task/03-date-tasks.js

```diff
@@ -93,8 +99,18 @@ function timeSpanToString(startDate, endDate) {
  *    Date.UTC(2016,3,5,18, 0) => Math.PI
  *    Date.UTC(2016,3,5,21, 0) => Math.PI/2
  */
-function angleBetweenClockHands(date) {
-    throw new Error('Not implemented');
+function angleBetweenClockHands(UTC_Date) {
```

Comment:
Why does an argument name start with an uppercase letter? JS uses `camelCase` as the coding convention